packages/components/src/Stack/Stack.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useStyles, system } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-flatten-children';
+
+type StackProps = {
+  space?: 0 | 4 | 8 | 16 | 24 | 32 | 40 | 48 | 56 | 64 | 88;
```

Comment:
Why doesn't it use `spaces` from the theme 🤔

Reply:
I thought we dont want a dependency from theme

Reply:
No variants, but will still have to make sure that everything adheres the spaces of the whole app.

Reply:
hm ok but if i import space from theme i have also a dependency to the b2b theme? Os there another way?

Reply:
That's not what I meant. The space printers should read from theme.space. there is no need to depend on a certain theme.

Reply:
Updated this PR :)